docs/running-on-coclobas.md

```diff
@@ -0,0 +1,131 @@
+# Running Epidisco With Ketrew/Coclobas
+
+## Setting Things Up
+
+This quickstart guide will get you running Epidisco on test data (or your own)
+on the GCloud platform, using our Kubernetes scheduler, Coclobas.
+
+To keep things as simple as possible, we use a script (`please.sh`) to do most
+of the work. If you're interested in what's going on behind the scenes, please
+check out the script, which is documented, or you can always try `./please.sh
+type <command>` to see what a particular command is doing, and just
+`./please.sh` to get a list of all possible commands.
+
+### Set Up a Launch Box
+
+We need a server on GCloud to run our scheduler and Ketrew, our workflow
+engine. This box will serve as the interface between your workflow and GCloud's running
+of them.
+
+You'll need the GCloud command line utility, installed
+[from here](https://cloud.google.com/sdk/downloads#interactive).
+
+Download our setup script and use it to create a GCloud box:
+
+```shell
+wget https://...
+chmod 777 disco.sh
+./disco.sh create example-box-name
+```
+
+Once it's created, you will be able to ssh in with `gcloud compute ssh
+example-box-name`. The rest of the instructions will be carried out inside this
+VM.
+
+
+### Configuring the Cluster
+
+We will now create an NFS filer that the cluster will use, and generate a
+configuration file for both a Coclobas cluster and, to be used later, a Biokepi
+machine. Note that `disco.sh` should have been copied to your home directory on
+this box for you already.
+
+```shell
+./disco.sh configure
+```
+
+TODO: how to get GATK_JAR_URL. MUTECT_JAR_URL, NET_MHC_jars...
+TODO: explain what's going on, and that the config file is editable by hand
+
+Next we want to set up and run our Docker image, which is where Coclobas and
+Ketrew will be running, and eventually is where we will submit our workflows
+from.
+
+```shell
+sudo bash ./disco.sh setup-docker
+sudo bash ./disco.sh enter-docker
+```
+
+### Inside the Docker
+
+```shell
+cd /coclo
+./disco.sh create-nfs 1000  # size in GB
+./disco.sh start-all
+```
+
+You should now be in a screen session with four windows.
+
+- `Ketrew-server`
+- `Coclobas-server`
+- `Sudo-tlstunnel`
+- `bash`
+
+You will be in the `bash` window.
+
+The first time the Coclobas server is started, it needs to create a the
+Kubernetes cluster. This may take about 5-10 minutes; you can check its status
+with `./disco.sh cluster-status`.
+
+If it says `Initializing` wait a bit and try again, if it says `Ready`, we can
+go on to the next step.
+
+You can visit the Google Container Engine WebUI watch your cluster being
+created in the "Instance Group" tab.
```

Comment:
I think it's better to use the REST API to query the status; who wants to open a browser?